src/js/jwt/jwt.js

```diff
@@ -218,10 +219,11 @@ function loginAllTabs() {
 /*** User Functions ***/
 // Get user information
 exports.getUserInfo = () => {
-    log('Getting User Information');
+    log('Getting User Information CHANGES!');
```

Comment:
Do we need this `CHANGES!` in the log?

Reply:
No i just totally forgot to remove it. I've already removed it.